layers/exporters/to_excel.py

```diff
@@ -33,15 +27,12 @@ def to_xlsx(self, layer, filepath="layer.xlsx"):
             :param layer: A layer to initialize the instance with
             :param filepath: The location to write the excel file to
         """
-        if layer is not None:
-            if not isinstance(layer, Layer):
-                raise TypeError
-            
-        if layer is None:
-            raise NoLayer
```

Comment:
Removed because layer is a required argument and therefore presumably doesn't require additional checking of preconditions aside from value.